app/models/ems_refresh/refreshers/container_refresher_mixin.rb

```diff
@@ -0,0 +1,38 @@
+module EmsRefresh
+  module Refreshers
+    module ContainerRefresherMixin
+      def kubernetes_fetch_inventory(client)
+        fetch_entities(client, KUBERNETES_ENTITIES)
+      end
+
+      def openshift_fetch_inventory(client)
```

Comment:
Regardless of the position or naming of the needs-to-be-somewhat-shared Kubernetes method, I don't see why anything containing the word "openshift" should exist outside its class hierarchy. :confused: